.github/workflows/integration.yml

```
@@ -226,26 +232,39 @@ jobs:
           script: |
             const log = github.log
             return {
+              runnerDebugMode: core.isDebug(),
               debug: log.debug === console.debug,
               info: log.info === console.info
             }
-      - run: |
-          echo "- Validating debug default"
-          expected='{debug:false,info:false}'
-          if [[ "${{steps.debug-default.outputs.result}}" != "$expected" ]]; then
-            echo $'::error::\u274C' "Expected '$expected', got ${{steps.debug-default.outputs.result}}"
-            exit 1
-          fi
-          echo "- Validating debug set to true"
-          expected='{debug:true,info:true}'
-          if [[ "${{steps.debug-true.outputs.result}}" != "$expected" ]]; then
-            echo $'::error::\u274C' "Expected '$expected', got ${{steps.debug-true.outputs.result}}"
-            exit 1
-          fi
-          echo "- Validating debug set to false"
-          expected='{debug:false,info:false}'
-          if [[ "${{steps.debug-false.outputs.result}}" != "$expected" ]]; then
-            echo $'::error::\u274C' "Expected '$expected', got ${{steps.debug-false.outputs.result}}"
-            exit 1
-          fi
+      - id: evaluate-tests
+        name: Evaluate test outputs
+        env:
+          RDMODE: ${{ (runner.debug == '1' && 'true') || 'false' }}
```

Can this also be `${{ runner.debug == '1' }}`?

```suggestion
          RDMODE: ${{ runner.debug == '1' }}
```